### ISSUE TITLE
[Keyboard Navigation – Make Code – About] 'About' pop-up window is not getting close via 'ESC' key.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3627]
**User Experience:**
User depending on Keyboard might get affected as user is unable to close the 'Dialog' using the "ESC " key which may hinder the user to use the application effectively.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "About" option and select it.
7. Navigate through all the controls appearing under "About" dialog.
8. Verify that 'About' pop-up window is not getting close via 'ESC' key.

**Actual Result:**
'About' pop-up window is not getting close via 'ESC' key.

**Expected Result:**
'About' pop-up window should be close via 'ESC' key.

**Note:**
1. Same issue exists on Firefox & Chrome.
2. Same issue exists under 'Report Abuse' pop-up.
3. Same issue exists under 'Pair device' pop-up.
4. same issue is exists throughout the application

**WCAG Reference:**
[WCAG 2.4.5 - Multiple Ways
](https://www.w3.org/WAI/WCAG21/Understanding/multiple-ways.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox, Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG2.4.5, #Win10-Edge(Chromium)

[WCAG2.4.5_About.zip](https://github.com/microsoft/pxt-microbit/files/5509672/WCAG2.4.5_About.zip)
